docs/source/inference/likelihood_inference.rst

```diff
@@ -0,0 +1,6 @@
+Maximum Likelihood
```

Comment:
This section is also inconsistent with the way that resources are cited.

Reply:
Also properly cited now.